requirements/dev.txt

```diff
@@ -12,5 +12,6 @@ sphinx>=2.0.0
 alabaster
 johnnydep
 deepdiff
+orderly-set==5.3.0; python_version == '3.8'
```

Comment:
just for python 3.8 or greater than 3.8?

Reply:
Just 3.8. See [this comment on orderly-set's PR that resolves this](https://github.com/seperman/orderly-set/pull/11#issuecomment-2802739146).